src/solvers/prop/prop.h

```diff
@@ -116,6 +116,15 @@ class propt:public messaget, public prop_assignmentt
     warning() << "CPU limit ignored (not implemented)" << eom;
   }
 
+  /// Set random variable frequency (Exclusive to minisat2)
```

Comment:
Also glucose has this feature. Output at least a warning if the solver does not implement this feature.

Reply:
Glucose build doesn't seem to want to work, I'm not sure which parameters to add. I'm skipping it. Maybe we can add this later.

Reply:
Not sure which problem you've run into, but https://github.com/marek-trtik/cbmc/commit/2548e7f30476dbd1909004d3864872b1b400f879 might be helpful.